### ISSUE TITLE
g_hostname variable not defined ?

### ISSUE THREAD

**Issue by cadencep45** [ImTheLucKyOne/check_mk_emcunity#11]
I see

File "/omd/sites/dev2/local/share/check_mk/checks/emcunity_fs", line 105, in check_emcunity_fs
    status, infotext, perfdata = df_check_filesystem_single(g_hostname, item, size_mb, avail_mb, 0, None, None, params)

when I check for definitions, I see

 grep g_hostname *
emcunity_fs:            status, infotext, perfdata = df_check_filesystem_single(g_hostname, item, size_mb, avail_mb, 0, None, None, params)
emcunity_lun:            status, infotext, perfdata = df_check_filesystem_single(g_hostname,    item, size_mb, avail_mb, 0, None, None, params)
emcunity_pool:            status, infotext, perfdata = df_check_filesystem_single(g_hostname, item, size_mb, avail_mb, 0, None, None, params)
emcunity_vmfs:            status, infotext, perfdata = df_check_filesystem_single(g_hostname, item, size_mb, avail_mb, 0, None, None, params)

g_hostname not defined.

grep size_mb *
emcunity_fs:            size_mb = int(size_B) / 1024.0 / 1024.0
emcunity_fs:            allocated_percent = (allocated_mb / size_mb) * 100
emcunity_fs:            status, infotext, perfdata = df_check_filesystem_single(g_hostname, item, size_mb, avail_mb, 0, None, None, params)
emcunity_lun:            size_mb = int(size_B) / 1024.0 / 1024.0
emcunity_lun:            avail_mb = size_mb - used_mb
emcunity_lun:            status, infotext, perfdata = df_check_filesystem_single(g_hostname,    item, size_mb, avail_mb, 0, None, None, params)
emcunity_pool:            size_mb = int(size_B) / 1024.0 / 1024.0
emcunity_pool:            sub_percent = (sub_mb / size_mb) * 100
emcunity_pool:            status, infotext, perfdata = df_check_filesystem_single(g_hostname, item, size_mb, avail_mb, 0, None, None, params)
emcunity_vmfs:            size_mb = int(size_B) / 1024.0 / 1024.0
emcunity_vmfs:            avail_mb = size_mb - used_mb
emcunity_vmfs:            status, infotext, perfdata = df_check_filesystem_single(g_hostname, item, size_mb, avail_mb, 0, None, None, params)

size_mb defined

grep avail_mb *
emcunity_fs:            avail_mb = int(avail_B) / 1024.0 / 1024.0
emcunity_fs:            status, infotext, perfdata = df_check_filesystem_single(g_hostname, item, size_mb, avail_mb, 0, None, None, params)
emcunity_lun:            avail_mb = size_mb - used_mb
emcunity_lun:            status, infotext, perfdata = df_check_filesystem_single(g_hostname,    item, size_mb, avail_mb, 0, None, None, params)
emcunity_pool:            avail_mb = int(avail_B) / 1024.0 / 1024.0
emcunity_pool:            status, infotext, perfdata = df_check_filesystem_single(g_hostname, item, size_mb, avail_mb, 0, None, None, params)
emcunity_vmfs:            avail_mb = size_mb - used_mb
emcunity_vmfs:            status, infotext, perfdata = df_check_filesystem_single(g_hostname, item, size_mb, avail_mb, 0, None, None, params)

avail_mb defined

should g_hostname  be defined somewhere ?

**Comment by ImTheLucKyOne** [comment]
If you are using cmk version 1.5.0 you have to go to the cmk 1.5.0 branch. 

https://github.com/ImTheLucKyOne/check_mk_emcunity/tree/cmk-1.5.0-compatibility

This is also stated in the readme btw.

**Comment by cadencep45** [comment]
Yep, when I went investigating I found our cmk admin had been gung-ho rather than checking version required.
